src/profile/UitpasCard/UitpasCard.tsx

```diff
@@ -47,15 +47,19 @@ const UitpasCard: FC<TProps> = ({ passHolder, isLarge }) => {
             <BrandLogo height={16} inverse />
           </Styled.LogoContainer>
           <Styled.BarcodeContainer isLarge={isLarge}>
-            <Barcode
-              format="CODE128"
-              height={isLarge ? 80 : 40}
-              singleBarWidth={isLarge ? 2 : 1.4}
-              value={firstActiveCard.uitpasNumber}
-            />
-            <Typography size={isLarge ? 'normal' : 'small'} topSpacing={isLarge ? '8px' : ''}>
-              {applyBarcodeMask(firstActiveCard.uitpasNumber)}
-            </Typography>
+            {firstActiveCard.uitpasNumber && (
```

Comment:
Zoude niet beter een eerste actieve card zoeken met uitpasnummer en die hier tonen, en dan al de andere kaarten wel tonen van naam, ongeacht of ze een uitpasnr hebben?